charts/app/values.yaml

```diff
@@ -15,9 +15,9 @@ appealReplyServiceApi:
     port: 3000
     logLevel: trace
     notify:
-      baseUrl: "" # Let Notify package resolve this itself
-      secretName: ""
-      serviceId: ""
+      baseUrl: '' # Let Notify package resolve this itself
+      secretName: ''
+      serviceId: ''
       emailReplyToId:
```

Comment:
My YAML formatter doesn't like your changes to this file.  Are you sure about these?

Reply:
Prettier kicked in..will revert all of that to be sure